src/apis/storage_partitioning.js

```diff
@@ -95,6 +95,28 @@ export const partitioningSetEncrypt = ({ encrypt, partitioning }) => {
             });
 };
 
+/**
+ * @param {string} partitioning     DBus path to a partitioning
+ */
+export const partitioningSetHomeReuse = async ({ partitioning }) => {
+    const request = await getPartitioningRequest({ partitioning });
+
+    // In general default scheme defined in conf.Storage.default-scheme would be used
+    // Only btrfs (AUTOPART_TYPE_BTRFS) autopartitioning scheme is supported now
+    request["partitioning-scheme"] = cockpit.variant("i", 1);
+    // These settings work also for AUTOPART_TYPE_LVM, AUTOPART_TYPE_LVM_THINP schemes
+    request["reformatted-mount-points"] = cockpit.variant("as", ["/"]);
+    request["reused-mount-points"] = cockpit.variant("as", ["/home"]);
+    request["removed-mount-points"] = cockpit.variant("as", ["/boot", "bootloader"]);
+
+    // Settings for AUTOPART_TYPE_PLAIN
```

Comment:
Please dont leave commented out code when this is reaching a final state.

Reply:
Fixed, all 4 schemes are supported internally now.